src/note.rs

```diff
@@ -245,7 +245,7 @@ impl Note {
     /// Defined in [Zcash Protocol Spec § 4.8.3: Dummy Notes (Orchard)][orcharddummynotes].
     ///
     /// [orcharddummynotes]: https://zips.z.cash/protocol/nu5.pdf#orcharddummynotes
-    pub(crate) fn dummy(
+    pub fn dummy(
```

Comment:
why?
generaly we do not want to expose this.
Consider a soution where the dummy padding is happaning inside orchard (add relevent function)

Reply:
I need dummy notes and keys to build a tx in librustzcash tests

Reply:
exposing an unneeded api publicly for the sake of external tests will not fly.
Can we reproduce the conntent of `dummy()` inside `librustzcash`?

Reply:
I would need to duplicate a lot, not sure its the best way. Perhaps we play with test-dependencies and features instead, what do you think?

Reply:
Dummy methods have been copied to librustzcash

Reply:
just make sure it is inside `#[Test]`